### DISCUSSION TITLE
[Q] Adding custom font to a text field

### DISCUSSION THREAD

**Discussion by ohtwadi** [strapi/strapi#6478] [Legacy Topics]
Hi,

I am a newbie to Strapi, would appreciate being pointed in the right direction. 

My task is to have a text field which would use a custom font. How can this be achieved?

**Comment by ESnark** [comment]
Strapi is a headless CMS which means It gives you tools for structuring content and I/O API for the content. The rest is up to a front-end developer.

..And a custom font belongs to "The rest". You can tell the frontend developer which font to use.

Another option is to [replace the wysiwyg editor](https://strapi.io/blog/how-to-change-the-wysiwyg-in-strapi). Existing one provides a markdown editor. Of course, custom fonts not included. The replacement takes more work but gives users more formatting options. However, even if you choose the last option, there is no guarantee that it will look the same on the client side. you need to set up rules with the front end developer.

**Comment by ohtwadi** [comment]
This is helpful. Thanks! I will consume more documentation. Please close.
